multicast/src/commonMain/kotlin/com/dropbox/flow/multicast/StoreRealActor.kt

```diff
@@ -0,0 +1,12 @@
+package com.dropbox.flow.multicast
+
+import kotlinx.coroutines.CoroutineScope
+
+internal expect abstract class StoreRealActor<T>(
```

Comment:
Why use expect/actual on this class?

Reply:
Currently it is backed by kotlin actor which is not compatible with native afaik

Reply:
Correct, now I remember.